### PR TITLE
kubie support added fixes #414

### LINKED ISSUE #414: Add kubie to TOOLS
## Expected Behaviour

Arkade get kubie would download kubie as a tool

## Current Behaviour

As of this moment arkade get doesn't have kubie in the list

## Possible Solution

download from https://github.com/sbstp/kubie/releases

## context for kubie:

kubie is an alternative to kubectx, kubens and the k on prompt modification script. It offers context switching, namespace switching and prompt modification in a way that makes each shell independent from others. It also has support for split configuration files, meaning it can load Kubernetes contexts from multiple files. You can configure the paths where kubie will look for contexts, see the settings section.

* What arkade version is this?

```
arkade version 0.7.15
```

## Are you a GitHub Sponsor (Yes/No?)

Check at: https://github.com/sponsors/alexellis
- [ ] Yes
- [X] No

### PR DESCRIPTION
**Pull request by aboe76**
## Description
kubie cli added

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

fixes #414 

## How Has This Been Tested?

```
$ podman container run --rm -v $(pwd):/workspace -ti golang:1.16.3-alpine sh
# cd /workspace
# go build
# ./arkade get kubie
```
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [X] I have signed-off my commits with `git commit -s`
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

**Comment by alexellis** [comment]
Hey @aboe76 thanks for the PR. Could you rebase please and push again?

See also: https://www.youtube.com/watch?v=8j0H6urZ-bU

File: pkg/get/get_test.go [hunk 1]
```diff
@@ -1504,3 +1504,43 @@ func Test_DownloandHelm(t *testing.T) {
...
 
 }
+
+func Test_DownloadKubie(t *testing.T) {
+	tools := MakeTools()
+	name := "kubie"
+
+	tool := getTool(name, tools)
+
+	tests := []test{
```
Comment (alexellis) [inline pkg/get/get_test.go:1514]: Given that there's no 32-bit ARM binary for Raspberry Pi OS, would you mind also raising an issue on https://github.com/sbstp/kubie and requesting it?

**Review by alexellis — CHANGES_REQUESTED** [review]
Approved, however a rebase is required.

Be careful not to create any merge commits or additional commits in your PR as this will cause a delay in merging.

File: pkg/get/get_test.go (hunk 1 above)
Comment (aboe76) [reply pkg/get/get_test.go:1594]: @alexellis, done asked for a 32bit release

**Comment by aboe76** [comment]
@alexellis the developer of kubie responded:

https://github.com/sbstp/kubie/issues/78#issuecomment-852608375

File: pkg/get/get_test.go (hunk 1 above)
Comment (aboe76) [reply pkg/get/get_test.go:1594]: kubie released version 0.15 with 32bit support

**Comment by alexellis** [comment]
Hi, we need a rebase on this please :pray:

**Comment by alexellis** [comment]
Hi @aboe76 I asked for a rebase 4 weeks ago and didn't hear back from you.

Would you like me to close this? Did you forget about it, or are you sending us a hint that you don't want to continue with the PR?

**Comment by aboe76** [comment]
@alexellis I'm sorry it took so long to rebase, it has slipped my mind, thanks for reminding me

File: pkg/get/get_test.go (hunk 1 above)
Comment (aboe76) [reply pkg/get/get_test.go:2194]: updated version to 0.15.1 with 32bit

**Pull request closed without merging**